sr_gui_grasp_controller/src/sr_gui_grasp_controller/grasp_controller.py

```diff
@@ -463,10 +458,13 @@ def __init__(self, context):
 
         selector_layout.addWidget(QtWidgets.QLabel("Select Hand"))
 
+        # TODO(@anyone): Make so grasp controller combo box can be used to select multiple hand H
         self.hand_combo_box = QtWidgets.QComboBox()
-
-        for hand_serial in self.hand_parameters.mapping.keys():
-            self.hand_combo_box.addItem(hand_serial)
+        hand_finder = HandFinder()
+        if hand_finder.hand_e_available():
+            hand_parameters = hand_finder.get_hand_parameters()
+            for hand_serial in self.hand_parameters.mapping.keys():
+                self.hand_combo_box.addItem(hand_serial)
 
         selector_layout.addWidget(self.hand_combo_box)
```

Comment:
Looks like you found something anyway :) (referring to your commet here: https://github.com/shadow-robot/sr-visualization/pull/176#pullrequestreview-39506473)

I made it so the drop down isn't displayed if the if statement doesn't run. I'll push these changes to the PR for this (see link) on branch: F#SRC-76/B_testing_grasp_controller